src/tests/ftest/checksum/basic_checksum.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+import os
+import ctypes
+import time
+import agent_utils
+import server_utils
+import write_host_file
+
+from pydaos.raw import (DaosPool, DaosContainer, IORequest,
+                        DaosObj)
+from apricot import TestWithServers
+
+
+class ChecksumContainerValidation(TestWithServers):
+    """
+    Test Class Description:
+
+    :avocado: recursive
+    """
+    # pylint: disable=too-many-instance-attributes
+    def setUp(self):
+        super(ChecksumContainerValidation, self).setUp()
+        self.agent_sessions = None
+        self.pool = None
+        self.container = None
+        self.obj = None
+        self.ioreq = None
+        self.hostlist = None
+        self.hostfile = None
+        self.no_of_dkeys = None
+        self.no_of_akeys = None
+        self.array_size = None
+        self.record_length = None
+        server_group = self.params.get("name",
```

Comment:
Special reason this is being explicitly defined? "self.server_group" from apricots test.py can also be used.

Reply:
Done. Removed it.